### PR TITLE
Migration guide from qiskit-ibmq-provider

### PR DESCRIPTION
**Pull request by 1ucian0**
I was unable to find a guide to migrate `qiskit-ibmq-provider` classes into the new `qiskit-ibm-runtime`. So I wrote a small equivalence table for the documentation.

It looks like this:
screencapture-file-Users-bel-code-qiskit-ibm-runtime-docs-build-html-migrate-from-ibmq-html-2023-02-07-14_10_19

### CODE REVIEW

**Review by kt474 — COMMENTED** [review]
Just minor grammar suggestions, otherwise LGTM - thanks!

File: docs/migrate_from_ibmq.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+.. list-table:: Migrate from ``qiskit.providers.ibmq.runtime`` in ``qiskit-ibmq-provider`` to ``qiskit-ibm-runtime`` 
+   :header-rows: 1
+
+   * - class in ``qiskit-ibmq-provider``
+     - class in ``qiskit-ibm-runtime``
+     - Notes
+   * - ``qiskit.providers.ibmq.runtime.IBMRuntimeService``
+     - :class:`qiskit_ibm_runtime.QiskitRuntimeService`
+     - ``IBMRuntimeService`` class was removed from ``qiskit_ibm_runtime 0.6.0`` and replaced by :class:`~QiskitRuntimeService`.
```
Comment (kt474) [inline docs/migrate_from_ibmq.rst:23]: Not super familiar with rst files but will :class:`~QiskitRuntimeService` render the link correctly?

Comment (HuangJunye) [reply docs/migrate_from_ibmq.rst:23]: If you use `intersphinx` extension it should work: https://qiskit.github.io/qiskit_sphinx_theme/sphinx_guide/how_to_cross-reference.html#pages-from-other-documentation-projects

Comment (1ucian0) [reply docs/migrate_from_ibmq.rst:23]: No worries.. I will go the traditional way.

**Review by kt474 — APPROVED** [review]

**Pull request merged**